src/components/Common/InputBox/ProfileImageInputBox/ProfileImageInputBox.jsx

```diff
@@ -1,22 +1,34 @@
+import { useState } from 'react'
+
 import s from './ProfileImageInputBox.module.scss'
 
+import { uploadImage } from 'api/image'
+import { PROFILE_IMAGE_ONLY_UPLOAD_IMAGE } from 'constants/SIGN_ERROR'
+import basicProfileImg from 'assets/img/basic-profile-img.svg'
 import uploadImg from 'assets/img/upload.svg'
 
-const ProfileImageInputBox = ({ image, setImage }) => {
-  const handleProfileImageUpload = e => {
+const ProfileImageInputBox = ({ setImage }) => {
+  const [profileImage, setProfileImage] = useState(basicProfileImg)
```

Comment:
여기도 초기값 설정 부분을 추가해주실수 있나요?
```suggestion
const ProfileImageInputBox = ({ setImage, initialImage ='basicProfileImg' }) => {
  const [profileImage, setProfileImage] = useState(initialImage)
```

Reply:
이미지를 빼먹었네요 추가하겠습니다~